pkgs/by-name/co/cosmic-term/package.nix

```
@@ -49,18 +49,20 @@ rustPlatform.buildRustPackage rec {
   '';
 
   nativeBuildInputs = [
-    cmake
-    just
-    pkg-config
-    makeBinaryWrapper
+     just
+     pkg-config
+     makeBinaryWrapper
```

```suggestion
    just
    pkg-config
    makeBinaryWrapper
```

Use 2 spaces of indentation.